sbin/common/sbom.sh

```diff
@@ -1,4 +1,32 @@
 #!/bin/bash
+# Build the CycloneDX Java library and app used for SBoM generation
+buildCyclonedxLib() {
+  local javaHome="${1}"
+
+  # Check if JDK-17 is available, download it if it isn't
+  if [ ! -d "${javaHome}" ]; then
+    # Download JDK-17
+    apiUrlTemplate="https://api.adoptium.net/v3/binary/latest/\${JDK_BOOT_VERSION}/\${releaseType}/linux/\${downloadArch}/jdk/hotspot/normal/\${vendor}"
```

Comment:
You don't need these 2 lines:
apiUrlTemplate=...
apiURL=$(eval echo ${apiUrlTemplate})